src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.scss"
```

Comment:
Stylesheets with the `.scss` extension cannot be linked directly in HTML. You need to compile SCSS to CSS and then link the compiled `.css` file.